AlphaWallet/TokenScriptClient/Models/AssetAttributeSyntaxValue.swift

```diff
@@ -272,8 +272,9 @@ extension Dictionary where Key == AttributeId, Value == AssetAttributeSyntaxValu
         self["street"]?.subscribableStringValue
     }
 
-    var traitsAssetInternalValueValue: AssetInternalValue? {
-        self["traits"]?.value
+    var traitsValue: [OpenSeaNonFungibleTrait]? {
+        guard case .openSeaNonFungibleTraits(let traits) = self["traits"]?.value else { return nil }
```

Comment:
Nice. But can you replace the guard with the original switch? Easier to discover breakage. Same reason for avoiding `default:` in the codebase.